### ISSUE TITLE
Gif/Giphy Block: Add Alt Text field

### ISSUE THREAD

**Issue by msilbers** [Automattic/wp-calypso#64750]
### What

Right now the Giphy block doesn't allow you to set alt-text anywhere, and it seems to pull alt-text from the image on giphy itself. Alt text should be contextual so this seems unexpected and is sometimes very lacking, and this should have an option for people to add their own alt-text related to their content.

### Why

This is an accessibility issue, all images and gifs should have proper alt-text, and with this block we can't set it.
